app/views/categories/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= stylesheet_link_tag 'signIn' %>
+<%= stylesheet_link_tag 'category_index' %>
+
+<div class="cat_header">
+ <div class="i_back"> <%= link_to "<", root_path, class:"back" %></div>
+ <div class="p_title">New Categories</div>
+ <div class="i_menu"><img src="https://icon-library.com/images/white-search-icon-png/white-search-icon-png-4.jpg" alt="No transactions_img" style="width: 150px;"></div>
+</div>
```

Comment:
- [ ] Dear @fredo509, You'r app functions well. But I have faced an issue while creating a category. It works fine when I initially create a category with valid input, but I test the app with an invalid input for the name and image URL, once I did that the app crashed and I could not see the categories list or create a new category. Please address this. Remember, the user will insert an invalid value to the input files of the form, and we need to validate user inputs and keep the app functional.

### The issue
https://github.com/fredo509/Budget_App/assets/69077061/06e8c290-a4dd-4722-9d8e-08609ea866da